src/Catty/Helpers/Util/Util.swift

```diff
@@ -70,6 +98,32 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         return deviceName
     }
 
+    class func look(withName objectName: String?, forObject object: SpriteObject?) -> Look? {
+        guard let lookList = object?.lookList else {
+            return nil
+        }
+
+        for look in lookList {
+            if let look = look as? Look, look.name == objectName {
+                return look
+            }
+        }
+
+        return nil
+    }
+
+    class func object(withName objectName: String?, forScene scene: Scene?) -> SpriteObject? {
```

Comment:
See my previous comment.

Reply:
I've changed the method signature following your suggestions.